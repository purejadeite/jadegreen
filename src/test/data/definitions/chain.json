{
	"sheets": [
		{
			"comment": "連結",
			"target": {
			    "name": "chain"
			},
		    "cells": [
		        {"id": "values", "row": 2, "options": [{"type": "chain", "keyId": "key", "chainId": "chain"}],
		            "columns": [
		                {"id": "chain", "column": 4, "splitter": "\n", "always": false, "options": [{"type": "remove"}]},
		                {"id": "key", "column": 2},
		                {"id": "value", "column": 3}
		            ]
		        }
		    ]
		}
	]
}