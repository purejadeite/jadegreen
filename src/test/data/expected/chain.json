[
    {
        "values": [
            {
                "key": "file1",
                "value": "ファイル1",
                "chain": [
                    {
                        "key": "file2a",
                        "value": "ファイル2Ａ",
                        "chain": {
                            "key": "file3",
                            "value": "ファイル3",
                            "chain": {
                                "key": "file4",
                                "value": "ファイル4"
                            }
                        }
                    },
                    {
                        "key": "file2b",
                        "value": "ファイル2Ｂ",
                        "chain": "file3"
                    }
                ]
            },
            {
                "key": "file5",
                "value": "ファイル5",
                "chain": {
                    "key": "file6",
                    "value": "ファイル6"
                }
            },
            {
                "key": "file7",
                "value": "ファイル7"
            },
            {
                "chain": "file8",
                "key": "file8",
                "value": "ファイル8"
            }
        ]
    }
]